packages/mdl-radio/mixin.js

```diff
@@ -0,0 +1,147 @@
+export const Class = {
+  CHECKED: 'mdl-radio--checked',
+  DISABLED: 'mdl-radio--disabled',
+  FOCUSED: 'mdl-radio--focused'
+};
+
+export const Identifier = {
+  ROOT: 'mdl-radio',
+  NATIVE_INPUT: 'mdl-radio__input'
+};
+
+// A unique ID counter to distinguish between radio buttons.
+let idCounter = 0;
+
+export default function MDLRadioMixin(renderer) {
+  Object.assign(this, {
+    initMdlRadio_(eventBus) {
+      this.renderer_ = bindAll(renderer, this);
+
+      this.eventBus_ = eventBus;
+
+      this.id = idCounter++;
```

Comment:
That being said, since we're dealing so heavily with DOM, should we choose something a bit more differentiating than `id` here for a name?